playwright-e2e/utils/test-utils.ts

```diff
@@ -249,7 +250,7 @@ export function studyShortName(study: StudyEnum): { shortName: string | null; re
       break;
     case StudyEnum.ANGIO:
       shortName = 'angio';
-      realm = 'Angio';
+      realm = dsmEnv === 'dsm-test' ? 'Pepper-Angio' : 'Angio';
```

Comment:
Test env domain is different.

<img width="857" alt="Screenshot 2023-12-06 at 11 07 41 AM" src="https://github.com/broadinstitute/ddp-angular/assets/35533885/b9f5892c-7231-4a2e-a074-6d0773fdefb7">